pkg/plugins/runtime/k8s/webhooks/injector/injector.go

```diff
@@ -256,10 +256,12 @@ func (i *KumaInjector) loadContainerPatches(
 		}
 	}
 
+	var missingPatches []string
+
 	for _, patchName := range patchNames {
 		containerPatch := &mesh_k8s.ContainerPatch{}
 		if err := i.client.Get(ctx, kube_types.NamespacedName{Namespace: i.systemNamespace, Name: patchName}, containerPatch); err != nil {
-			logger.Error(err, "could not get a ContainerPatch")
+			missingPatches = append(missingPatches, patchName)
```

Comment:
Should we check if `err` is NotFoundError?

Reply:
addressed